### ISSUE TITLE
Not Recognizing a post was autoflagged

### ISSUE THREAD

**Issue by K-Davis1** [Charcoal-SE/metasmoke#526]
[MS autoflagged](https://metasmoke.erwaysoftware.com/post/142922/flag_logs) [a post](https://metasmoke.erwaysoftware.com/post/142922), using four flags, on a post that was actually just an **naa**.  Upon submitting my naa feedback, I observed that there were autoflags and realized that a chat message wasn't provided to ping the users whose flags were used. I switched my feedback to **fp**, which came with the same result. ([transcript](https://chat.stackexchange.com/transcript/11540?m=47550221#47550221)).

I then used the chat message `autoflagged`, in which SD responded with:

> _@K.Dᴀᴠɪs That post was not automatically flagged by metasmoke._

Due to this bug, it then made sense that upon me providing my feedback that there was no automatic chat message sent to ping these users as MS doesn't recognize there were any flags.

Afterwards on a [few other occasions](https://chat.stackexchange.com/search?q=SmokeDetector+autoflagged&user=223524&room=11540), I submitted the same chat message to SD on different posts that were also auto-flagged and received the same response (which in these cases were all tpu's). 

This leads me to believe that at the present time - any autoflagged post with fp feedback will no longer notify users to retract their flags.

**Comment by makyen** [comment]
The issue of the chat `autoflagged` command has been fixed.

However, there's definitely an issue with ping notification upon FP feedback on an autoflagged post. I just gave FP feedback, as the first feedback, to [this SD report](https://chat.stackexchange.com/transcript/message/47586288#47586288), but no pings were made.

As an associated issue, there used to be, probably still is, an problem where the pings would not happen unless the FP was the first feedback. In other words, if it got a TP feedback and then an FP feedback there would be no pings. However, that's a separate, but associated, issue/problem.
